### DISCUSSION TITLE
Welcome to jDeploy Discussions!

### DISCUSSION THREAD

**Discussion by shannah** [shannah/jdeploy#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ChrisOlsen195** [comment]
Hello, Everyone --
    My name is Chris Olsen, retired college professor (statistics), and new to jDeploy.

    I just installed a program on my Mac (Intel chip) and it worked like a charm.  Now, however, I am trying to uninstall the program (professorial curiosity) and have run into a problem -- I can't get rid of it!    The program does not appear in the Applications folder, which I expected.  When I bring up the terminal and go to the Applications folder I see it there, but interestingly I don't see anything else in the Applications folder!   (I do see a long list of programs in the Desktop Applications folder.)   I tried rm [file name] and the terminal assured me that there is no such program there.   Any clue what I might be doing and/or what I should be doing?  (I am not an accomplished Mac  person...)

    TIA!

**Reply by shannah** [reply]
You'll find the program in your ~/Applications directory.  That is the "Applications" directory inside your user home directory, which is different than the global Applications directory.

**Comment by ChrisOlsen195** [comment]
Hi, Steve --

   Found it!   Thank you...Loving jDeploy!

   -- Chris
